src/olympia/devhub/templates/devhub/index-legacy.html

```diff
@@ -0,0 +1,149 @@
+    </div>
+    <div class="col-2">
+      <div class="island sidebar-breakout">
+        {% if recent_addons %}
+            <h2>{{ _('My Add-ons') }}</h2>
+
+            <div id="my-addons">
+              {% for item in recent_addons %}
+                <div class="addon-item">
+                  <div class="addon-details">
+                    <p>
+                      <a class="addon-name" href="{{ item.addon.get_dev_url('edit') }}">
+                        <img class="addon-icon" src="{{ item.addon.icon_url }}" alt="">
+                        <strong>{{ item.addon.name }}</strong>
+                      </a>
+                    </p>
+                    <p>
+                      <strong>{{ _('Status:') }}</strong>
+                      <span class="{{ status_class(item.addon) }}"><b>{{ item.addon.STATUS_CHOICES[item.addon.status] }}</b></span>
+                    </p>
+                    <p>
+                      <strong>{{ _('Visibility:') }}</strong>
+                      {% if not item.addon.is_disabled and item.addon.is_listed %}
+                        {{ _('Listed') }}
+                      {% elif item.addon.is_disabled and item.addon.is_listed %}
+                        {{ _('Hidden') }}
+                      {% elif not item.addon.is_listed %}
+                        {{ _('Unlisted') }}
+                      {% endif %}
+                    </p>
+                    {% if item.addon.current_version %}
+                      <p>
+                        <strong>{{ _('Latest Version:') }}</strong>
+                        {{ link_if_listed_else_text(item.addon.current_version,
+                                         item.addon.current_version.version) }}
+                      </p>
+                    {% endif %}
+                    {% with position = item.position %}
+                      {% if position and position.pos and position.total %}
+                        <p>
+                          <strong>{{ _('Queue Position:') }}</strong>
+                          {% trans position=position.pos|numberfmt,
+                                   total=position.total|numberfmt %}
+                            {{ position }} of {{ total }}
+                          {% endtrans %}
+                        </p>
+                      {% endif %}
+                    {% endwith %}
+                    {% if not item.addon.is_persona() and item.addon.has_complete_metadata() and not item.addon.is_disabled %}
+                      <p class="upload-new-version">
+                        <a href="{{ item.addon.get_dev_url('versions') }}#version-upload">
+                          {{ _('Upload New Version') }}</a>
+                      </p>
+                    {% endif %}
+                  </div>
+                </div>
+              {% endfor %}
+            </div>
+            <div class="flex-spring"></div>
+            <p><a id="link-view-all" href="{{ url('devhub.addons') }}">{{ _('View all add-ons ►') }}</a></p>
```

Comment:
That glyph seems like it'd be better as a CSS element and not in the translation, but maybe I'm wrong.